docs/3.0-migration-guide.md

```diff
@@ -13,7 +13,25 @@ The changes are detailed below, and include:
 - Removal of many insecure or obsolete features
 - Tidying up of configuration options (including removing some less useful options).
 - Changing function signatures, e.g. adding return codes, adding extra parameters, or making some arguments const.
-- Removal of functions previously marked as deprecated.
+- Removal of functions, macros, and types previously marked as deprecated.
+
+
+## Referring to Mbed TLS 2.x documentation
```

Comment:
This section title seems a bit strange. Yes, the contents of the section explain how to access (by generating) the 2.x docs, but it's not until reading this section that we learn _why_ we might want to do that.

Given that section headings are often used to find bits we need to read, it's easy to miss that this might be needed.

How about adding a one-line paragraphy after the final bullet point ("Removal of functions...")

```
Much of the information needed to determine a migration path can be found in the Mbed TLS 2.x documentation.
```

and changing this section heading to

```
## Accessing the Mbed TLS 2.x documentation
```